From 6c696c90e845e0792a9c48db6a5ce9209e01137a Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Fri, 11 Feb 2022 19:40:47 +0200
Subject: [PATCH 10/11] plat: bl31: Replace obsolete S32G_HAS_HV with new name

The S32G_HAS_HV define has been removed and replaced with
S32_HAS_HV as part of adding support for ATF on S32R, when
the S32G-configs were replaced with more generic alternatives.

A occurrence of S32G_HAS_HV escaped, this commit fixes it.

Issue: ALB-7411
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32_lowlevel_bl31.S | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plat/nxp/s32/s32_lowlevel_bl31.S b/plat/nxp/s32/s32_lowlevel_bl31.S
index ff87b364f..4b44d087a 100644
--- a/plat/nxp/s32/s32_lowlevel_bl31.S
+++ b/plat/nxp/s32/s32_lowlevel_bl31.S
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019-2021 NXP
+ * Copyright 2019-2022 NXP
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
@@ -111,7 +111,7 @@ func s32g_gic_fixups_for_secondary
 	mrs	x8, SCR_EL3
 	/* ...clear NS and possibly HCE */
 	mov	x9, #SCR_NS_BIT
-#if (S32G_HAS_HV == 0)
+#if (S32_HAS_HV == 0)
 	orr	x9, x9, #SCR_HCE_BIT
 #endif
 	mvn	x9, x9
-- 
2.17.1

